octicons/octicons.less

```diff
@@ -39,7 +36,9 @@
 .octicon-arrow-small-right:before { content: '\f071'} /*  */
 .octicon-arrow-small-up:before { content: '\f09f'} /*  */
 .octicon-arrow-up:before { content: '\f03d'} /*  */
-.octicon-beer:before { content: '\f069'} /*  */
+.octicon-microscope:before,
```

Comment:
I've left the microscope as an alias for the beaker for now to help migrate things. We can eventually remove that if it turns out it isn't needed.